packages/create-bison-app/template/cypress/plugins/index.ts

```diff
@@ -68,11 +41,11 @@ export default (on, _config) => {
     disconnectDB: () => {
       return disconnect();
     },
-    factory: ({ name, attrs }) => {
-      const Factory = Factories[`${name}Factory`];
+    factory: ({ name, attrs }: { name: string, attrs: any }) => {
+      const Factory = (Factories as any)[`${name}Factory`];
```

Comment:
How intense do you want the type annotations to be here? If you really wanted to, you could get rid of the `any` by using this as the type for the `name` parameter:

```ts
type FactoryNames = keyof typeof Factories extends `${infer T}Factory` ? T : never;
```

Of course, I doubt it would add that much value - it doesn't add autocompletion to the places where this function is called. So feel free to skip it.

Reply:
Great suggestion! Updated.